deployments/run-juno.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# <UDF name="run_juno_testnet" label="Run Juno node on testnet" default="./build/juno --network 0" example="./build/juno --network 0" />
+# <UDF name="run_juno_mainnet" label="Run Juno node on mainnet" default="./build/juno --network 1" example="./build/juno --network 1" />
+
+
+exec >/root/SSout 2>/root/SSerr
+sudo apt-get update
+
+JUNO_TESTNET_CMD="${RUN_JUNO_TESTNET:-./build/juno --network 0}"
+# codenoid
+# https://gist.github.com/codenoid/4806365032bb4ed62f381d8a76ddb8e6
+printf "Checking latest Go version...\n";
+LATEST_GO_VERSION="$(curl --silent https://go.dev/VERSION?m=text)";
+LATEST_GO_DOWNLOAD_URL="https://golang.org/dl/go1.18.linux-amd64.tar.gz"
```

Comment:
Are these two vars used for something? Looks like was used to always install the latest Go version but now version 1.18 is hardcoded